bin/build-styles

```diff
@@ -10,7 +13,36 @@ const inputFile = './src/assets/css/styles.scss';
 const outputFile = './build/blog/assets/css/styles.css';
 const isProduction = process.env.ELEVENTY_ENV === 'production';
 
+const buildAddonsFrontendBlogUtils = async function () {
+  const addonsFrontendBlogUtilsInputFile = path.resolve(
+    'node_modules/addons-frontend-blog-utils/style.css'
+  );
+  const addonsFrontendBlogUtilsOutputFile = path.resolve(
+    'build/blog/assets/css/addons-frontend-blog-utils.style.css'
+  );
+  const css = await fs.readFile(addonsFrontendBlogUtilsInputFile, 'utf8');
+  // addons-frontend-blog-utils style.css has paths to images, we want to copy
+  // those to the right directory, rewrite the stylesheet, then the rewritten
+  // file can be imported from ours.
+  const options = {
+    url: 'copy',
+    basePath: path.resolve('node_modules/addons-frontend-blog-utils'),
+    // assetsPath is supposed to be relative to `to` or absolute, but it
+    // doesn't seem to work correctly with absolute paths or with '../img', so
+    // we set it to be empty and allow the images to live inside 'css/'.
+    assetsPath: '',
```

Comment:
This isn't great, but it works... I may be doing things the wrong way, but I couldn't get `assetsPath` to work like it should with absolute paths (or relative paths with `../`)

Reply:
Found a fix in https://github.com/mozilla/addons-blog/pull/415/commits/7c689df9847427910a5e5918dd78d9dcec1441fb